README.md

```diff
@@ -34,7 +34,7 @@
 
 # 📖 Financier <a name="about-project"></a>
 
-**Financier** is a Rails application for managing expenses and keeping track of transactions.
+**Financier** is a Rails application for managing expenses and keeping track of transactions. This allows them to save more
```

Comment:
- [ ] As per the project [design guidelines](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#design), for the splash screen page, Please add the correct background color which is `white`, add the correct font color and weight for the app title, and add the correct placement for the `login` and `sign up` buttons.

https://github.com/Samuelwanza/financier/blob/f30decf7b5075dcd5ee2651363a40aefc07b633d/app/views/splash/index.html.erb#L2-L6

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/Samuelwanza/financier/assets/68796232/1eeccb5d-9d33-43d6-95af-7070c261554a)  | ![image](https://github.com/Samuelwanza/financier/assets/68796232/b48efd01-76ec-40fc-847f-07b43d2aec63)  |